src/System.Private.CoreLib/src/System/Text/Normalization.Windows.cs

```diff
@@ -24,7 +24,7 @@ public static bool IsNormalized(this string strInput, NormalizationForm normaliz
             Interop.mincore.SetLastError(Interop.ERROR_SUCCESS);
```

Comment:
It looks odd to combine explicit SetLastError PInvoke with Marshal.GetLastWin32Error.